docs/src/components/MigrationChanges/ForgotPasswordRenameAlert.tsx

```diff
@@ -0,0 +1,50 @@
+import { Alert, Text } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+import {
+  Framework,
+  FRAMEWORK_DISPLAY_NAMES,
+  MAJOR_VERSIONS,
+} from '../../data/frameworks';
+
+export const ForgotPasswordRenameAlert = ({ framework }) => {
+  const {
+    query: { platform = 'react' },
+  } = useRouter();
+
+  if (!framework) {
+    framework = platform as Framework;
+  }
+
+  const isReactNative = framework === 'react-native';
+  const prevFrameworkVersion = MAJOR_VERSIONS[framework][1];
```

Comment:
We should probably hardcode these prevFramework versions rather than making them the second item in the array. Won't this end up being wrong if we bump another MV version?

Reply:
Yeah I can hardcode those. I was thinking it might be more extensible as an array bc as long as it's ordered the latest would be first and prev second, but if we're not going to include a list of all major versions probably doesn't make sense to keep a list.